examples/dt/dcn/service-values.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
     config.kubernetes.io/local-config: "true"
 data:
   preserveJobs: false
+  cinder:
+    uniquePodNames: false
```

Comment:
Ok, looks good to me. I see we are actually enabling this parameter by default in [1], and not just for NFS (unibeta).
I'm ok to minimize the amount of changes, and good to have `uniquePodNames` enabled by default, so we can catch issues earlier.
+1 for me.
 
[1] https://github.com/openstack-k8s-operators/architecture/blob/main/lib/control-plane/openstackcontrolplane.yaml#L61